apps/web/components/BlockRecommendedProducts/BlockRecommendedProducts.vue

```diff
@@ -39,149 +39,95 @@
+          >
+            <SfIconCheck
+              class="mr-1 w-[1.1rem]"
+              :class="{ invisible: recommendedBlockValue.text.textAlignment !== 'right' }"
             />
+            Right
           </div>
         </div>
       </div>
-      <div class="mb-6">
-        <UiFormLabel class="mb-1">Text Alignment (y)</UiFormLabel>
-
-        <Multiselect
-          v-model="recomandedBlockValue.text.textAlignment"
-          :options="['left', 'center', 'right']"
-          placeholder="Select align"
-          :allow-empty="false"
-          class="cursor-pointer"
-          select-label=""
-          deselect-label="Selected"
-        />
-      </div>
-
-      <div class="mb-6">
-        <UiFormLabel class="mb-1">Textbox Alignment (x)</UiFormLabel>
-
-        <Multiselect
-          v-model="recomandedBlockValue.text.justify"
-          :options="['top', 'center', 'bottom']"
-          placeholder="Select justify"
-          :allow-empty="false"
-          class="cursor-pointer"
-          select-label=""
-          deselect-label="Selected"
-        />
-      </div>
-
-      <div class="mb-6">
-        <UiFormLabel class="mb-1">Textbox Alignment (y)</UiFormLabel>
-
-        <Multiselect
-          v-model="recomandedBlockValue.text.align"
-          :options="['left', 'center', 'right']"
-          placeholder="Select an alignment"
-          :allow-empty="false"
-          class="cursor-pointer"
-          select-label=""
-          deselect-label="Selected"
-        />
-      </div>
     </div>
   </div>
 </template>
 
 <script setup lang="ts">
-import { clamp } from '@storefront-ui/shared';
 import { SfInput, SfTextarea } from '@storefront-ui/vue';
-import Multiselect from 'vue-multiselect';
 const { data, updateProductRecommendedProductsOptions } = useHomepage();
+const { blockIndex } = useSiteConfiguration();
 
-const recomandedBlock = computed(
-  () =>
-    (data.value.blocks.find((block: Block) => block.name === 'ProductRecommendedProducts')?.options ||
-      {}) as ProductRecommendedProductsOptions,
+const recommendedBlock = computed(
+  () => (data.value.blocks[blockIndex.value].options || {}) as ProductRecommendedProductsOptions,
 );
 
-const recomandedBlockValue = reactive({
-  ...recomandedBlock.value,
-  text: recomandedBlock.value.text || {
+const recommendedBlockValue = reactive({
+  ...recommendedBlock.value,
+  text: recommendedBlock.value.text || {
     pretitle: '',
     title: '',
     subtitle: '',
     htmlDescription: '',
     color: '#000',
-    bgcolor: '#fff',
-    bgopacity: 1,
     textAlignment: 'left',
-    justify: 'center',
-    align: 'left',
   },
 });
-
-const clampBrightness = (event: Event, type: string) => {
-  const currentValue = (event.target as HTMLInputElement)?.value;
-  const nextValue = Number.parseFloat(currentValue);
-  recomandedBlockValue.text.bgopacity = clamp(nextValue, 0, 1);
-};
+recommendedBlockValue.text.color = recommendedBlockValue.text.color || '#000';
 
 watch(
```

Comment:
I don't think the watcher is necessary? My understanding is that the `computed` above handles all the updates.